protocol/testing/e2e/gov/bridge_test.go

```diff
@@ -101,7 +118,43 @@ func TestUpdateProposeParams(t *testing.T) {
 			},
 			expectedProposalStatus: govtypesv1.ProposalStatus_PROPOSAL_STATUS_PASSED,
 		},
-		"Fail: invalid authority": {
+		"Failure: negative propose delay duration": {
+			msg: &bridgetypes.MsgUpdateProposeParams{
+				Authority: authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+				Params: bridgetypes.ProposeParams{
+					MaxBridgesPerBlock:           genesisProposeParams.MaxBridgesPerBlock,
+					ProposeDelayDuration:         -genesisProposeParams.ProposeDelayDuration,
+					SkipRatePpm:                  genesisProposeParams.SkipRatePpm,
+					SkipIfBlockDelayedByDuration: genesisProposeParams.SkipIfBlockDelayedByDuration,
+				},
+			},
+			expectCheckTxFails: true,
+		},
+		"Failure: negative skip if block delayed by duration": {
```

Comment:
Since you wrote the above tests with values that case the tests cases to (mostly) check to see if any field on EventParams was modified with an invalid update, do you think it's valuable to try that here too? (Not a requirement, just a thought. It looks like these tests are written a bit differently.)

Reply:
yeah for any test case where `expectedProposalStatus` is not `passed`, the test checks that params are unchanged